pep-0625.rst

```diff
@@ -39,17 +39,17 @@ predictability to assign this format certain contextual information that helps
 the installation process. pip, for example, parses the file name of an sdist
 from a :pep:`503` index, to obtain the distribution's project name and version
 for dependency resolution purposes. But due to the lack of specification,
-the installer does not have any guarantee to the correctness of the inferred
+the installer does not have any guarantee as to the correctness of the inferred
 message, and must verify it at some point by locally building the distribution
 metadata.
 
-This build step is awkward for a certin class of operations, when the user
+This build step is awkward for a certain class of operations, when the user
 does not expect the build process to occur. `pypa/pip#8387`_ describes an
 example. The command ``pip download --no-deps --no-binary=numpy numpy`` is
 expected to only download an sdist for numpy, since we do not need to check
 for dependencies, and both the name and version are available by introspecting
 the downloaded file name. pip, however, cannot assume the downloaded archive
-follows the convention, and must build check the metadata. For a :pep:`518`
+follows the convention, and must check the metadata. For a :pep:`518`
```

Comment:
I think I intended to write “build and check” here.

Reply:
Fixed!